cinch/roles/certificate_authority/handlers/main.yml

```diff
@@ -1,2 +1,2 @@
 - name: install certificate authority
-  command: update-ca-trust
+  command: update-ca-trust force-enable
```

Comment:
What does this "force-enable" option do? On my version of update-ca-trust, that command is not document. The only documented command is either an empty command or the "extract" command.